pkg/platform/runtime/runtime.go

```diff
@@ -63,11 +75,33 @@ func newRuntime(target setup.Targeter, an analytics.Dispatcher, svcModel *model.
 		}
 	}
 
+	if target.ProjectDir() == "" || auth == nil {
+		return rt, nil
+	}
```

Comment:
Auth shouldn't matter as it should b e optional for the buildplan API. If it's not then that needs to be addressed, but I'm fairly certain it is as we have already had a lot of back and forth over this with extensibility.

Reply:
I was going by this line: https://github.com/ActiveState/cli/blob/mitchell/dx-1861/pkg/platform/model/buildplanner.go#L94
I will add an `auth != nil &&` bit to the conditional in order to allow for optional auth.